### ISSUE TITLE
Need more criteria than just a list

### ISSUE THREAD

**Issue by quanghoc** [mmccaff/PlacesToPostYourStartup#57]
It'd be better to put down submission criteria and timeframe for EACH website for better planning. For example:
1. Must be in WA (Geek Wire)
2. Must be private (BetaList)
3. Must not be launched several weeks already (BetaList)
4. 3-4 weeks for acceptance
etc...

**Comment by mmccaff** [comment]
I agree that a written submission criteria could be beneficial if it were something lightweight that didn't take a lot of time to evaluate.

There is a Travis test that validates that the links in the PR are at least HTTP 200, and I manually check additions, but it can be subjective and it'd be nice to be able to point to "criteria" if something were to be not accepted for submission.

**Comment by karllhughes** [comment]
I think some indicator of the popularity of each would be helpful too. For example, it would be nice to know the Alexa ranking of these sites.

There is a [somewhat unofficial API](https://stackoverflow.com/a/6224304) (it's undocumented, but works with no limits as it's for their browser toolbar) for Alexa data that looks like this:

http://data.alexa.com/data?cli=10&dat=snbamz&url=betalist.com

or 

http://data.alexa.com/data?cli=10&dat=snbamz&url=producthunt.com

So we could theoretically check these endpoints for each URL and see which ones are the most popular sites.

**Comment by rebootcode** [comment]
Alexa ranking is not the right away to assume site's trust.

It's better to check Domain Authority via Moz.com or any other Available resources!

Anything above 40 should be considered as an acceptable list!

**Comment by karllhughes** [comment]
@rebootcode I was actually more interested in helping startups prioritize where they post rather than using this metric as a minimum threshold for inclusion, but that's not a bad idea either.

Either way, if this list included more data, I think it should probably be reformatted as a table (similar to how it's done with [public-apis](https://github.com/toddmotto/public-apis)).
